app/src/pages/inside/common/dashboardPageHeader/dashboardPageHeader.scss

```diff
@@ -0,0 +1,27 @@
+.dashboard-page-header {
+
+}
+
+.dashboards-nav-list-mobile {
+  display: none;
+  @media (max-width: $SCREEN_XS_MAX) {
+    display: block;
+    padding: 13px 10px;
+    background: $COLOR--white-two;
+    span {
+      a {
```

Comment:
Is it really necessary to write nested tag selectors?

Reply:
Added `activeClassName` to `NavLink`